test/unittests/parser_test.cpp

```diff
@@ -661,15 +661,15 @@ TEST(parser, global_float)
     EXPECT_FALSE(module.globalsec[0].type.is_mutable);
     EXPECT_EQ(module.globalsec[0].type.value_type, ValType::f32);
     EXPECT_EQ(module.globalsec[0].expression.kind, ConstantExpression::Kind::Constant);
-    EXPECT_EQ(Value(module.globalsec[0].expression.value.constant).f32, 1.2f);
+    EXPECT_EQ(module.globalsec[0].expression.value.constant.f32, 1.2f);
     EXPECT_TRUE(module.globalsec[1].type.is_mutable);
     EXPECT_EQ(module.globalsec[1].type.value_type, ValType::f64);
     EXPECT_EQ(module.globalsec[1].expression.kind, ConstantExpression::Kind::Constant);
-    EXPECT_EQ(Value(module.globalsec[1].expression.value.constant).f64, 3.4);
+    EXPECT_EQ(module.globalsec[1].expression.value.constant.f64, 3.4);
     EXPECT_FALSE(module.globalsec[2].type.is_mutable);
     EXPECT_EQ(module.globalsec[2].type.value_type, ValType::f64);
     EXPECT_EQ(module.globalsec[2].expression.kind, ConstantExpression::Kind::GlobalGet);
-    EXPECT_EQ(module.globalsec[2].expression.value.constant, 1);
+    EXPECT_EQ(module.globalsec[2].expression.value.global_index, 1);
```

Comment:
Hm, this was a typo iirc, we did not fix it in another PR? Will make sure this has its own commit.

Reply:
We actually have its own commit already.